widget/embedded/src/components/HeaderButtons/HomeButtons.tsx

```diff
@@ -50,25 +44,24 @@ export function HomeButtons(props: HomeButtonsPropTypes) {
       </Tooltip>
 
       {!isNotificationsHidden && (
-        <Tooltip
+        <Popover
+          align="center"
+          collisionBoundary={layoutRef}
+          collisionPadding={{ right: 20, left: 20 }}
           container={getContainer()}
-          side="top"
-          content={i18n.t('Notifications')}>
-          <Popover
-            align="center"
-            collisionBoundary={layoutRef}
-            collisionPadding={{ right: 20, left: 20 }}
-            container={getContainer()}
-            content={<NotificationContent />}>
-            <HeaderButton
-              size="small"
-              variant="ghost"
-              onClick={onClickNotifications}>
-              <NotificationsIcon size={18} color="black" />
-              <UnreadNotificationsBadge />
-            </HeaderButton>
-          </Popover>
-        </Tooltip>
+          content={<NotificationContent />}>
+          <div>
```

Comment:
I fixed it in another way.
what do you think?
https://github.com/rango-exchange/rango-client/pull/488/files